hangman.js

```diff
@@ -1 +1,125 @@
-console.log("welcome to hangman");
\ No newline at end of file
+var game = {
+  word: [],
+  hidden: [],
+  guesses: 5,
+
+  getWord: function(){
+    var temp = window.prompt("Have someone enter a secret word for you to begin the game.");
+    this.word = temp.toLowerCase().split("");
+    this.hidden = Array(this.word.length).fill(" - ");
+  },
+
+  getGuess: function(){
+    var message = "Word: " + this.hidden.join("");
+    message += "\nYou have " + this.guesses + " guesses.";
+    message += "\nEnter a letter.";
+
+    return window.prompt(message);
+  },
+
+  checkWin: function(){
+    if(this.word.toString() == this.hidden.toString())
```

Comment:
what's the difference between `.toString()` and `.join()`, if not significant, recommend picking one and being consistent.